core/CoreTypes.hpp

```diff
@@ -9,29 +9,23 @@ namespace olympia::core_types
     using RegisterBitMask = sparta::Scoreboard::RegisterBitMask;
 
     //! Register file types
-    enum RegFile : uint8_t {
+    enum RegFile : uint8_t
+    {
         RF_INTEGER,
         RF_FLOAT,
+        RF_VECTOR,
         RF_INVALID,
         N_REGFILES = RF_INVALID
     };
 
-    static inline const char * const regfile_names[] = {
-        "integer",
-        "float"
-    };
+    static inline const char* const regfile_names[] = {"integer", "float", "vector"};
 
-    static inline const char * const issue_queue_types[] = {
-        "alu",
-        "fpu",
-        "br"
-    };
+    static inline const char* const issue_queue_types[] = {"alu", "fpu", "br", "vint", "vset"};
```

Comment:
Since we support multiple types of vector pipes (`vint`, `vmul`, `vdiv`, etc.) I would change `vint` to just `vec`.

Reply:
This was actually dead code, not used anymore since we define issue queue types through `.yaml`. Removed in [73ca9f4](https://github.com/riscv-software-src/riscv-perf-model/pull/167/commits/73ca9f4adb0bce0b8756a9b1ab3d88bf96c589bd)